front/public/locales/en/stdcm-simulation-report-sheet.json

```
@@ -12,6 +12,7 @@
   "from": "from",
   "infrastructure": "Infra",
   "length": "length",
+  "loadingGauge": "Loading Gauge",
```

Other fields start with a lower case (same for french translation)
```suggestion
  "loadingGauge": "loading gauge",
```

---

Done and moved to stdcm.json after rebase